README.md

```diff
@@ -44,6 +44,26 @@ Other forms of documentation:
 - For offline use, the documentation can be built from the source code.
   See the file [`doc/README.rst`](doc/README.rst)
 
+## Code of Conduct
```

Comment:
IMO having these items in a bulleted list would be less bewildering to users landing on the README. We could collect them under a single, appropriate heading.

Whichever way we organize, doc/index.rst should be updated with the same content. Most/many users will first land on that page, not on GitHub/the README.

Reply:
Do you think it's enough for `doc/index.rst` to state something like "If you're interested in our security policy, please refer to ``SECURITY.md``"? 
Because copying the contents directly would add a lot of "clutter" to our index page, but the above doesn't provide a link and people would still need to understand where to look for that file. Another workaround might be to include files like `SECURITY.rst` in `doc/` and make that link to `SECURITY.md`.

Reply:
I would do something like:
```
See our Security Policy `on GitHub <https://github.com/iiasa/message_ix/blob/...>`_ or in :file:`SECURITY.md` included with the source code.
```

This gives both a direct link and (for those offline, I guess increasingly rare these days) the file name.